flipperdevices/flipperzero-firmware
# Picopass write

# What's new

- Improvements to detecting cards that don't use "standard" key
- Add support to write to "standard" keyed picopass

# Verification 

- load a previously saved picopass
- select write
- place on another picopass that uses standard iClass key
- see success message
- read new card and see same credential as before

# Checklist (For Reviewer)
